### PR TITLE
Refactor confirm department page to use fixtures

### PR DESCRIPTION
**Pull request by kentsanggds**
## What

Refactored functional tests on confirm department to use fixtures

## How to review

exec run-tests

### CODE REVIEW

File: tests/functional/test_confirm_department.py [hunk 1]
```diff
@@ -1,28 +1,34 @@
...
 
 
+@pytest.mark.usefixtures('live_server')
 class When_first_visiting_confirm_department(object):
 
-    def it_has_yes_selected(self, live_server, browser):
-        browser.visit(url_for('main.confirm_dept', _external=True))
+    @pytest.fixture(autouse=True)
+    def setup_page(
+            self, browser, email_address, submit_known_email_address):
```
Comment (andyhd) [inline tests/functional/test_confirm_department.py]: `browser` is not used in this method

File: tests/functional/test_confirm_department.py [hunk 2]
```diff
@@ -1,28 +1,34 @@
...
+        submit_known_email_address(email_address)
 
+    def it_has_yes_selected(self, browser):
         assert browser.find_by_css('#confirm-0').checked
 
     def it_goes_to_idp_interstitial_when_continue_clicked(
-            self, live_server, browser):
-        browser.visit(url_for('main.confirm_dept', _external=True))
-
+            self, browser):
```
Comment (andyhd) [inline tests/functional/test_confirm_department.py]: This line doesn't need to be broken

File: tests/functional/test_confirm_department.py [hunk 3]
```diff
@@ -1,28 +1,34 @@
...
+@pytest.mark.usefixtures('live_server')
 class When_on_confirm_department_clicking_no(object):
 
-    def it_goes_to_confirm_email_when_continue_clicked(
-            self, live_server, browser):
-        browser.visit(url_for('main.confirm_dept', _external=True))
+    @pytest.fixture(autouse=True)
+    def setup_page(
+            self, browser, email_address, submit_known_email_address):
+        submit_known_email_address(email_address)
```
Comment (andyhd) [inline tests/functional/test_confirm_department.py:26]: `browser` is not used in this method

File: tests/functional/test_confirm_department.py [hunk 4]
```diff
@@ -1,28 +1,34 @@
...
 
+@pytest.mark.usefixtures('live_server')
 class When_on_confirm_department_clicking_no(object):
 
-    def it_goes_to_confirm_email_when_continue_clicked(
-            self, live_server, browser):
-        browser.visit(url_for('main.confirm_dept', _external=True))
+    @pytest.fixture(autouse=True)
+    def setup_page(
+            self, browser, email_address, submit_known_email_address):
```
Comment (andyhd) [inline tests/functional/test_confirm_department.py]: This line does not need to be broken

File: tests/functional/test_confirm_department.py (hunk 1 above)
Comment (andyhd) [reply tests/functional/test_confirm_department.py]: This line does not need to be broken

File: tests/functional/test_confirm_department.py [hunk 5]
```diff
@@ -1,28 +1,34 @@
+import pytest
 from flask import url_for
```
Comment (andyhd) [inline tests/functional/test_confirm_department.py:1]: Imports should be in alphabetical order within the groupings of core, 3rd party and local modules.

**Review by andyhd — CHANGES_REQUESTED** [review]

File: tests/functional/test_confirm_department.py (hunk 5 above)
Comment (kentsanggds) [reply tests/functional/test_confirm_department.py:1]: ok, thought imports were before from statements, will update

File: tests/functional/test_confirm_department.py (hunk 1 above)
Comment (kentsanggds) [reply tests/functional/test_confirm_department.py]: ok

File: tests/functional/test_confirm_department.py (hunk 2 above)
Comment (kentsanggds) [reply tests/functional/test_confirm_department.py]: ok

**Review by andyhd — APPROVED** [review]

**Pull request merged**